src/graphql/admin/types/object/account.ts

```diff
@@ -20,6 +23,12 @@ const Account: GraphQLObjectType<Account> = GT.Object<Account>({
     level: { type: GT.NonNull(AccountLevel) },
     status: { type: GT.NonNull(AccountStatus) },
     title: { type: GT.String },
+    wallets: {
+      type: GT.NonNullList(Wallet),
+      resolve: async (source) => {
+        return WalletsRepository().listByAccountId(source.id)
```

Comment:
I know some resolvers use services directly but we should avoid this. We need to implement a method to support this at app layer level

Reply:
It looks like we already have this method in the app layer, and I will use that instead.